modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/raft/MetaStorageWriteHandler.java

```diff
@@ -278,4 +309,50 @@ boolean beforeApply(Command command) {
 
         return false;
     }
+
+    private static class IdempotentCommandCache {
+        @Nullable
+        final Serializable result;
+
+        final HybridTimestamp commandStartTime;
```

Comment:
I don't think that we need HybridTimestamp here. Common long with current time should be enough. WDYT?

Reply:
I can replace it with long, but why current time? Command start time is hybrid time

Reply:
E.g. because we don't need HybridTimestampHere. Up to you. Not that important.